app/services/analytics_events.rb

```diff
@@ -3303,7 +3303,6 @@ def idv_in_person_usps_proofing_results_job_enrollment_incomplete(
   # @param [String] enrollment_code
   # @param [String] enrollment_id
   # @param [Float] minutes_since_established
-  # @param [Boolean] fraud_suspected
```

Comment:
I don't follow why this is being removed here.

Reply:
I removed this because it was a field being pulled from the USPS enrollment proofing response and in my usage of this method the response is not present. So I was getting a missing parameter error when calling this method without fraud_suspected. It seemed like the usage of this method pointed to this param as being optional because the fraud_suspected is being pulled from the proofing response, but maybe it should be based off the profile as well?

I will double check, maybe it should be based off both the USPS response and profile?

Reply:
@shanechesnutt-ft I’m curious to hear if you find any indication that the `fraud_suspected` value should be based off both the USPS response and profile.  If `fraud_suspected` is only based off the USPS response, maybe `fraud_suspected` should default to false when there is no USPS response present?  What do you two think about that approach?

Reply:
I don't see any indication that the fraud_suspected should be based off of both. I just see that the fraud_suspected is based off the USPS response. Do you think it feels optional? I am fine with defaulting it to false when response is not present in the GetUspsProofingResultsJob.

Reply:
Re-reading this, I worry my comment came across as slightly rude. It wasn't meant to! It was literally just me not being clear on the reason.

I agree that it's taken from the USPS response and should only represent that: https://github.com/18F/identity-idp/blob/main/app/jobs/get_usps_proofing_results_job.rb#L653

TBH the matrix of events, where they're called, and what data is available in them is a little confusing to me. If we never have `fraud_suspected` available it obviously makes sense to remove. But as I'm reading it, this is called from `log_enrollment_updated_analytics` which does accept `response`, so it should at least sometimes be present for this event. I think?

Looking in CloudWatch, this value seems to be null on expired enrollments (reason = "Enrollment has expired"), and is otherwise a Boolean. It is almost always false, though (probably since most people trying to commit identity theft are reluctant to do so in person in a federal building). That seems like a reasonable default to me.

Reply:
In lines with the reasoning Matt shared below

>  If we never have fraud_suspected available it obviously makes sense to remove. But as I'm reading it, this is called from log_enrollment_updated_analytics which does accept response, so it should at least sometimes be present for this event.

I would vote against removing it.  I would be fine with avoiding the missing parameter error by putting a default value of false.

Reply:
I decided to default it to nil at the analytic event method level. Just in case we get responses without fraud_suspected... not sure how likely that is though.